### DISCUSSION TITLE
Compilation of jinx-mod.so failed

### DISCUSSION THREAD

**Discussion by TriAttack238** [minad/jinx#192] [Q&A]
I'm on Linux mint trying to get this package to work for Emacs 29.4. In accordance with being an Ubuntu-based system I have installed both `libenchant-2-dev` and `pkgconf` onto my system, yet the package still does not compile. 

```
jinx-mod.c:19:10: fatal error: enchant.h: no such file or directory
```

Am I getting the name wrong? Additionally, I know for sure that usr/include/enchant-2/enchant.h definitely exists because I'm looking at it in my file manager at this very moment.

**Comment by minad** [comment]
That's really odd. Afaik both Mint and Ubuntu are close or derived from Debian (testing), which I use and where Jinx compiles without problems. I suggest you try to compile the module manually.
